operator-common/src/main/java/io/strimzi/operator/common/Util.java

```diff
@@ -539,4 +542,16 @@ public static String fromAsciiBytes(byte[] bytes) {
         return new String(bytes, StandardCharsets.US_ASCII);
     }
 
+    /**
+     * Build basic HTTP authentication header value.
+     *
+     * @param username Username.
+     * @param password Password.
+     *
+     * @return Header value.
+     */
+    public static String buildBasicAuthValue(String username, String password) {
```

Comment:
I think you should create some `CruiseControlUtils` to keep some order in what is used where.

Reply:
ok, will come handy in the future